build.gradle

```diff
@@ -6,9 +6,9 @@ buildscript {
 }
 
 plugins {
-    id 'com.android.application' version '7.3.1' apply false
-    id 'com.android.library' version '7.3.1' apply false
-    id 'org.jetbrains.kotlin.android' version '1.7.21' apply false
+    id 'com.android.application' version '8.1.0' apply false
+    id 'com.android.library' version '8.1.0' apply false
```

Comment:
Bumping the Android Gradle Plugin here to major version 8 [requires JDK 17](https://developer.android.com/build/releases/gradle-plugin#:~:text=Android%20Gradle%20plugin%208.1.0%20is%20a%20major%20release%20that%20includes%20a%20variety%20of%20new%20features%20and%20improvements.)